src/test/java/pages/SongsPage.java

```diff
@@ -11,13 +11,15 @@ public SongsPage (WebDriver givenDriver) {
         super(givenDriver);
     }
     @FindBy(css= ".song-item")
-    WebElement listAllSongs;
+    private WebElement listAllSongs;
     @FindBy(css=".song-item")
-    WebElement firstSongInSongList;
+    private WebElement firstSongInSongList;
     @FindBy(css=".playback")
-    WebElement playBtnFromList;
+    private WebElement playBtnFromList;
     @FindBy(css="[data-testid='sound-bar-play']")
-    WebElement progressBar;
+    private WebElement progressBar;
+    @FindBy(css = ".songs")
+    private WebElement allSongs;
 
     public SongsPage chooseAllSongList() {
```

Comment:
We can make the xpath or css the class variable (Locator)

![image](https://user-images.githubusercontent.com/104761482/219476701-1730ed47-6bc3-4435-a954-ae63a6d03dc0.png)

Reply:
yes I tried that but I get an error
![Screenshot 2023-02-16 172441](https://user-images.githubusercontent.com/123651329/219510847-92bec6c6-ca97-4a2c-b932-666b4820e416.png)